analytics/analytics/core/models.py

```diff
@@ -81,6 +81,10 @@ class Job(models.Model):
     job_size = models.CharField(max_length=128, null=True)
     stack = models.CharField(max_length=128, null=True)
 
+    unnecessary = models.BooleanField(
+        default=False, help_text="Whether this job has 'No need to rebuild' in its trace."
+    )
```

Comment:
@scottwittenburg Is there a more domain specific term for this?

Reply:
Hmmm :thinking:  

If you were thinking of any specific alternatives, please suggest.  Otherwise, I think `unnecessary` is fine.